features/frameworks/COMPONENT_FPGA_CI_TEST_SHIELD/source/MbedTester.cpp

```diff
@@ -26,23 +26,24 @@
 
 #define mbed_tester_printf(...)
 
-#define PHYSICAL_PINS           128
-#define LOGICAL_PINS            8
-#define FIRMWARE_SIZE           2192012
-#define FIRMWARE_REGION_SIZE    0x220000
-#define FIRMWARE_HEADER_SIZE    0x10000
-#define FLASH_SECTOR_SIZE       0x1000
-#define LENGTH_SIZE             0x4
-#define CRC_SIZE                0x4
-#define FLASH_SPI_FREQ_HZ       2000000
-#define ANALOG_COUNT            4
-
-#define PHYSICAL_NC    ((MbedTester::PhysicalIndex)0xFF)
-
-static const uint8_t KEY[8] = {
-    0x92, 0x9d, 0x9a, 0x9b,
-    0x29, 0x35, 0xa2, 0x65
-};
+namespace {
+    const auto physical_pins = 128;
+    const auto logical_pins = 8;
+    const auto firmware_region_size = 0x220000;
+    const auto firmware_header_size = 0x10000;
+    const auto flash_sector_size = 0x1000;
+    const auto length_size = 0x4;
+    const auto fixed_crc_size = 0x4;
+    const auto flash_spi_freq_hz = 2000000;
+    const auto analog_count = 4;
+    
+    const auto physical_nc = (MbedTester::PhysicalIndex)0xFF;
```

Comment:
Wouldn't it be simple to just write :
```suggestion
    const MbedTester::PhysicalIndex physical_nc = 0xFF;
```

Also, this special value might be better placed next to the typedef of PhysicalIndex in the header file.

Reply:
Done